Import component in component

helloimela opened the pull request:
- Add helpers.js to import components as a module in project
- Add function to automatically import sub components that exist inside component's template
- helpers.js is available in the dist folder

How to use in project:
Import `defineCustomElements` function :
`import { defineCustomElements } from 'node_modules/corporate-ui/dist/collection/helpers';`
Define custom elements needed in the project : 
`defineCustomElements(['c-header','c-footer','c-theme']);`

Fixing issue #32

Commit:
Merge branch 'master' into import_component_in_component

Commit:
Merge branch 'master' into import_component_in_component

Commit:
Merge branch 'master' into import_component_in_component

Commit:
Added helpers to import components in project

helloimela commented:
Should be compiled with webpack, separated issue created #45

Commit:
Delete .npmrc

Commit:
Remove stencil generated files

Commit:
Merge branch 'master' into import_component_in_component

awinny commented on src/helpers.js:
```diff
@@ -0,0 +1,33 @@
+import { defineCustomElement as defineCE } from '../esm/es5/corporate-ui.core'
```
As we are now exporting "defineCustomElements", the as syntax is no longer needed

helloimela replied:
That's correct...will change it and add a demo in Storybook

awinny commented:
Stange that the prettifier did not run with same settings as mine.
Can be seen in stencil.config.ts
Seems to be the same when it comes to the tslint, for example: "if (comp[0] === name) val = comp" should have been rendered with curly braces.

awinny commented:
We should also have a demonstration for this I think.
As a start we can use it in the storybook setup.

Commit:
Merge branch 'master' into import_component_in_component

Commit:
Improve code for import helpers
- Add possibility to render 'all' elements
- Removed 'as' syntax
- Demo added in storybook
